### PR TITLE
DOC Document new FormSchemaController

### PR DESCRIPTION
**Pull request by GuySartorelli**
## Issue
- https://github.com/silverstripe/silverstripe-admin/issues/1762

### CODE REVIEW

File: en/02_Developer_Guides/02_Controllers/07_CMS_JSON_APIs.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ summary: Creating standardised JSON APIs for authenticated users in the CMS.
 
 This document contains a standard set of conventions to be used when creating JSON APIs in the CMS that are used in conjunction with AJAX requests from authenticated CMS users.
 
-To view an example of a controller that follows these standards see [`LinkFieldController`](https://github.com/silverstripe/silverstripe-linkfield/blob/4/src/Controllers/LinkFieldController.php).
+To view an example of a controller that follows these standards see [`LinkFieldController`](api:SilverStripe\LinkField\Controllers\LinkFieldController).
```
Comment (ChloeSartorelli) [inline en/02_Developer_Guides/02_Controllers/07_CMS_JSON_APIs.md:10]: Unrelated change - link should be an API link

File: en/08_Changelogs/6.0.0.md [hunk 2]
```diff
@@ -780,7 +791,8 @@ Note that the change from `ViewableData` to `ModelData` specifically was made to
 
 ### GraphQL removed from the CMS {#graphql-removed}
 
-> [!INFO] If you need to use GraphQL in your project for public-facing frontend schemas, you can still install and use the [`silverstripe/graphql`](https://github.com/silverstripe/silverstripe-graphql) module.
+> [!NOTE]
+> If you need to use GraphQL in your project for public-facing frontend schemas, you can still install and use the [`silverstripe/graphql`](https://github.com/silverstripe/silverstripe-graphql) module.
```
Comment (ChloeSartorelli) [inline en/08_Changelogs/6.0.0.md:795]: Unrelated - wasn't rendering correctly.

File: en/08_Changelogs/6.0.0.md [hunk 3]
```diff
@@ -606,14 +606,25 @@ See the [full list of removed and changed API](#api-removed-and-changed) to see
...
 
-As a result of this change, to following classes are now direct subclasses of `AdminController`:
+The new [`FormSchemaController`](api:SilverStripe\Admin\FormSchemaController) class (which is a subclass of `AdminController`) now owns the logic required for injecting and managing forms inside modals.
 
-- [`SudoModeController`](api:SilverStripe\Admin\SudoModeController) - used to be a subclass of `LeftAndMain`.
-- [`CMSExternalLinksController`](api:SilverStripe\ExternalLinks\Controllers\CMSExternalLinksController) - used to be a direct subclass of [`Controller`](api:SilverStripe\Control\Controller).
+As a result of these changes, the following classes have had their class hierarchy updated:
+
+|class|old superclass|new superclass|
```
Comment (emteknetnz) [inline en/08_Changelogs/6.0.0.md]:
```suggestion
|Class|Old super class|New super class|
```

Comment (emteknetnz) [reply en/08_Changelogs/6.0.0.md]: Would parent class be better than super class?

**Review by emteknetnz — CHANGES_REQUESTED** [review]

File: en/08_Changelogs/6.0.0.md (hunk 3 above)
Comment (ChloeSartorelli) [reply en/08_Changelogs/6.0.0.md]: The capitalisation isn't consistent with other tables in this page. Can you please look at the page in its rendered form (so you can see in context) and decide if you want the headers for all tables capitalised or lower-case? If you want them capitalised I'll update all the other tables as well.

> Would parent class be better than super class?

They mean the same thing so I don't mind either way - just let me know whether you want that change or not.

We do already use "superclass" multiple times in the changelog, "super class" once, and "parent class" only once, in case that helps you decide.

Comment (emteknetnz) [reply en/08_Changelogs/6.0.0.md]: They should all be capitalized, looks wrong if it's lower case. Leave as super class if it's consistent

Comment (ChloeSartorelli) [reply en/08_Changelogs/6.0.0.md]: I think it looks weird capitalized lol but I'll make the change.

Comment (ChloeSartorelli) [reply en/08_Changelogs/6.0.0.md]: Done
